### DISCUSSION TITLE
Notes show reblogs with media

### DISCUSSION THREAD

**Discussion by gnomecore** [AprilSylph/XKit-Rewritten#308] [Suggestion Box]
Okay, so I doubt that this is possible but I wanted to suggest it anyway just in case:

What if reblogs with _only added media and no text_ showed up in the notes? (Image for clarity)

![example tumblr notes](https://user-images.githubusercontent.com/90390851/136667839-5539dce6-a3bd-4124-a300-4d55440946c4.png)

I imagine this could be useful if you are asking followers to reblog with an image (i.e. a specific meme you were looking for) and you could see which reblogs have added media.

That said, there would probably have to be something written into tumblr's own code so that this would be possible? And I doubt it is. But you never know!

**Comment by arcanetrivia** [comment]
I  have no idea if that's possible either, but it's a neat idea.

**Comment by AprilSylph** [comment]
If Tumblr's API exposed media in the post notes endpoint, Tumblr would already display it.

Given that I've seen at least one Tumblr engineer express the wish to build this into the web interface, and the fact the mobile apps already display media in notes, it's safe to assume this is on Tumblr's roadmap already. You may want to contact Tumblr support or send an ask to wip.tumblr.com to confirm this, though.

It would vaguely be possible to do this client-side by querying every individual reblog note for added media, but it isn't feasible, since it would
1. fire up to fifty requests at once just by opening a post's notes (and another <=50 every time you scroll to load new notes), which could easily considered spam
2. would not integrate with the conversation view as this concept demonstrates
